client/rest-high-level/src/test/java/org/elasticsearch/client/ml/job/process/ModelSizeStatsTests.java

```diff
@@ -31,6 +31,8 @@
     public void testDefaultConstructor() {
         ModelSizeStats stats = new ModelSizeStats.Builder("foo").build();
         assertEquals(0, stats.getModelBytes());
+        assertEquals(0, stats.getModelBytesExceeded());
+        assertEquals(0, stats.getModelBytesMemoryLimit());
```

Comment:
These two will change to `null` if the other changes I recommended are made.  So that makes the values set by the default constructed builder inconsistent.  But this is better than filling in values that didn't really exist in a JSON document.